linq4j/src/main/java/org/apache/calcite/linq4j/tree/Expressions.java

```diff
@@ -2791,6 +2791,13 @@ public static ThrowStatement throw_(Expression expression) {
     return new ThrowStatement(expression);
   }
 
+  /**
+   * Creates Statements with Raise Notice Statement.
```

Comment:
change it to Creates a Statement instead of  Creates Statements

Reply:
DONE.